WooCommerce/Classes/App Intents/CreateOrderAppIntent.swift

```diff
@@ -11,7 +11,7 @@ struct CreateOrderAppIntent: AppIntent {
     @MainActor
     func perform() async throws -> some IntentResult {
         MainTabBarController.presentOrderCreationFlow()
-        ServiceLocator.analytics.track(event: WooAnalyticsEvent.AppIntents.shortcutWasOpened(with: .createOrder))
+        ServiceLocator.analytics.track(event: .AppIntents.shortcutWasOpened(with: .createOrder))
```

Comment:
A bit of a nitpick, but these removals of the full specifier make it a little more difficult to review, and AFAIK don't contribute to the change. Were they needed when `WooAnalyticsEvent` was in WooFoundation, earlier in this PR's life?

Reply:
Yup these removals were from the previous `WooAnalyticsEvent` move to the WooFoundation framework and aren't required for the latest state of the PR. As I couldn't find a trivial way to revert these like how I can use regex replace for the imports, plus I think it makes the code simpler, I'll keep these removals 😅